pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/ClusterDataImpl.java

```diff
@@ -412,7 +412,20 @@ public ClusterDataImpl build() {
      *
      * @throws IllegalArgumentException exist illegal property.
      */
-    public void checkPropertiesIfPresent() throws IllegalArgumentException {
+     public void checkPropertiesIfPresent() throws IllegalArgumentException {
+        boolean hasServiceUrl = getServiceUrl() != null;
+        boolean hasServiceUrlTls = getServiceUrlTls() != null;
+        boolean hasBrokerServiceUrl = getBrokerServiceUrl() != null;
+        boolean hasBrokerServiceUrlTls = getBrokerServiceUrlTls() != null;
+
+        if (!hasServiceUrl && !hasServiceUrlTls) {
+                throw new IllegalArgumentException("Atleast one of ServiceUrl or ServiceUrlTls must be set.");
```

Comment:
At least

Reply:
Also I think we can use `StringUtils.isEmpty(getServiceUrl()) && StringUtils.isEmpty(getServiceUrlTls())` to reduce the codes

Reply:
@harshithasudhakar

Reply:
Yes, I'll make the necessary changes.

Reply:
> Also I think we can use StringUtils.isEmpty(getServiceUrl()) && StringUtils.isEmpty(getServiceUrlTls()) to reduce the codes

+1